apps/ledger-live-desktop/tests/component/modal.component.ts

```diff
@@ -100,4 +100,13 @@ export class Modal extends Component {
       isVisible = await element.isVisible();
     }
   }
+
+  @step("Fill amount")
+  async fillAmount(amount: string) {
```

Comment:
do we have amount field on every modal?

Reply:
Nop but at least on 2, delegate and send